certbot/cli.py

```diff
@@ -867,14 +867,20 @@ def prepare_and_parse_args(plugins, args, detect_defaults=False):  # pylint: dis
         metavar="DOMAIN", action=_DomainsAction, default=[],
         help="Domain names to apply. For multiple domains you can use "
              "multiple -d flags or enter a comma separated list of domains "
-             "as a parameter. (default: Ask)")
+             "as a parameter. The first provided domain will be used in "
+             "filenames and some software user interfaces unless otherwise "
```

Comment:
what filenames?
I find this a little too ambiguous to be comfortable with it coming out of --help